README.md

```diff
@@ -150,6 +150,18 @@ in `form_with`.
 
 `form_with` has some important differences compared to `form_for` and `form_tag`, and these differences apply to `bootstrap_form_with`. A good summary of the differences can be found at: https://m.patrikonrails.com/rails-5-1s-form-with-vs-old-form-helpers-3a5f72a8c78a, or in the [Rails documentation](api.rubyonrails.org).
 
+
+## Configuration
```

Comment:
I'd like to give a little more information to the reader around configuration. I suggest something like a first paragraph that says:

> `bootstrap_form` can be used out-of-the-box without any configuration. However, `bootstrap_form` does have an optional configuration file at `config/initializers/bootstrap_form.rb` for setting options that affect all generated forms in an application.
>
> 
> The current configuration options are:
> 
> * `default_form_attributes` `bootstrap_form` versions 4.4.0 and older added a `role="form"` attribute to all forms. The W3C validator will raise a warning on forms with a `role="form"` attribute. 

The actual example will depend on the path we choose for maintaining compatibility for version 4, and enabling the new default for version 5 (which will be the Bootstrap v5 version). See my comments on the pull request.